go/p2p/api/api.go

```diff
@@ -27,10 +27,31 @@ const (
 	TopicKindTx TopicKind = "tx"
 )
 
+// Status is the P2P status of a node.
+type Status struct {
+	// ID is the public key used for P2P communication.
+	ID signature.PublicKey `json:"id"`
```

Comment:
Should we just call this a public key?
```suggestion
	// PubKey is the public key used for P2P communication.
	PubKey signature.PublicKey `json:"pub_key"`
```